api/api/src/main/scala/uk/ac/wellcome/platform/api/services/WorksRequestBuilder.scala

```diff
@@ -153,7 +153,9 @@ object WorksRequestBuilder extends ElasticsearchRequestBuilder {
             field = "data.otherIdentifiers.value",
             values = identifiers)
         )
-
+      case filter @ AccessStatusFilter(_, _) =>
+        filter.toElasticQuery(
```

Comment:
We could just have a method here that converts the `IncludeExcludeFilter` into the query, rather than having it as part of the model to keep the separation clear, or at least consistent.